pkg/oc/cli/cmd/debug.go

```diff
@@ -269,7 +269,8 @@ func (o *DebugOptions) Complete(cmd *cobra.Command, f kcmdutil.Factory, args []s
 
 	// TODO: below should be turned into a method on PrintFlags
 	if (o.PrintFlags.OutputFormat != nil && len(*o.PrintFlags.OutputFormat) > 0) ||
-		(o.PrintFlags.TemplatePrinterFlags != nil && o.PrintFlags.TemplatePrinterFlags.TemplateArgument != nil) {
+		(o.PrintFlags.TemplatePrinterFlags != nil && o.PrintFlags.TemplatePrinterFlags.TemplateArgument != nil &&
+			len(*o.PrintFlags.TemplatePrinterFlags.TemplateArgument) > 0) {
```

Comment:
@deads2k Updated this to also take into account when a template arg has been specified. Was making debug.sh fail

Reply:
> @deads2k Updated this to also take into account when a template arg has been specified. Was making debug.sh fail

break these apart into bool assignments so I can read them here without the nesting.

Reply:
Done